### PR TITLE
Add timestamps and make logs more readable

### PR DESCRIPTION
**Pull request by krdln**

### CODE REVIEW

**Comment by krdln** [comment]
* [Example log with stripped garbage](http://logs.opencontrail.org/winci/30bdcc9858e44a199ef9f6ea9600b757/log.txt)
* [The same log on Jenkins](http://10.84.12.26:8080/job/WinContrail/job/winci-server2016-devel/111/console)

File: CIScripts/Build/Zuul.ps1 [hunk 1]
```diff
@@ -15,7 +15,6 @@ function Clone-ZuulRepos {
         "--zuul-url=$ZuulUrl",
         "--zuul-branch=$ZuulBranch",
         "--map=./CIScripts/clonemap.yml",
-        "--verbose",
         $GerritUrl
```
Comment (m-kostrzewa) [inline CIScripts/Build/Zuul.ps1:18]: We don't understand why this was removed, could you clarify?

File: Jenkinsfile [hunk 2]
```diff
@@ -10,11 +10,18 @@ def testEnvFolder
 
 def testbeds
 
+def cleanAnsiColors(String sourceFile, String outputFile) {
```
Comment (m-kostrzewa) [inline Jenkinsfile]: Maybe this should be moved to a separate groovy file

File: Jenkinsfile [hunk 3]
```diff
@@ -10,11 +10,18 @@ def testEnvFolder
...
+    def stripAnsiConceal = /sed -e 's|\x1b\[8[^m]*m[^\x1b]*\x1b\[[^m]*m||g'/
+    def stripEscapeCode  = /sed -e 's|\x1b\[[^m]*m||g'/
```
Comment (m-kostrzewa) [inline Jenkinsfile]: MK: could you add comments showing examples of how these regexes work? Like example strings that they match

**Review by m-kostrzewa — CHANGES_REQUESTED** [review]
Do timestamps appear in the logfiles on logs.opencontrail.org?

File: CIScripts/Build/Zuul.ps1 (hunk 1 above)
Comment (krdln) [reply CIScripts/Build/Zuul.ps1:18]: To get rid of ZUUL's `Debug` messages (`Info` messages are still present)

**Review by m-kostrzewa — APPROVED** [review]
Signed off by: MKos, JI.

**Comment by m-kostrzewa** [comment]
@krdln before you merge please remove ticket number from PR title, we decided to not use them going forward

**Pull request merged**